ofprotocol/ofp13/ofp13_parser_test.go

```diff
@@ -6230,3 +6230,55 @@ func TestSerializeSetAsync(t *testing.T) {
 		t.Error("Serialized binary of OfpSetAsync is not equal to expected value.")
 	}
 }
+
+func verifySerializedMessage(t *testing.T, expect []byte, msg OFMessage) {
+	actual := msg.Serialize()
+	e_str := hex.EncodeToString(expect)
+	a_str := hex.EncodeToString(actual)
+	if len(expect) != len(actual) || e_str != a_str {
+		t.Log("Expected Value is : ", e_str)
+		t.Log("Actual Value is   : ", a_str)
+		t.Error("Serialized binary is not equal to expected value.")
+	}
+}
+
+/*
+I create an OVS switch named s1 and use the following command to get the data
+send from the command line ovs-ofct to the switch :
+sudo strace -xx -s 16000 -e trace=sendto ovs-ofctl -Oopenflow13  add-flow s1 "cookie=0x401,table=1,priority=7000,in_port=1,vlan_tci=0x0000/0x0fff,actions=push_vlan:0x8100,set_field:4097->vlan_vi
+d,goto_table:2" 2>&1 > /dev/null | grep sendto | awk -F'"' 'BEGIN{nd=0;}{nd++;if (nd == 4) print $2}' | sed 's%\\x%,0x%g'
+*/
+func TestFunctionalAddVlan(t *testing.T) {
```

Comment:
What is purpose of this test ?
I think this is regression test for ActionSetField to confirm gofc can serialize ActionSetField correctly using OxmVlanVid which is different OxmField type from existing test case for ActionSetField. 
In my opinion,  the name of test case should represent what you want to do with that function, and function have to be implemented according to that name .
If you want to test only AddVlan case, I feel it is not good,  because it is not exhaustive.

And more, the naming convention is different with other functions.
In gofc, test function name is TestSerialize/TestParse<TARGET_STRUCTURE_NAME>.
I would be happy if you follow this rule as much as possible.